src/tools/attack/attack.cpp

```diff
@@ -27,14 +27,16 @@
 #define ATTACK_PORT_DEFAULT 443
 
 static QUIC_DATAPATH* Datapath;
+static PacketWriter* Writer;
 
 static uint32_t AttackType;
 static const char* ServerName;
 static const char* IpAddress;
 static QUIC_ADDR ServerAddress;
-static const char* Alpn = "h3-31";
 static uint64_t TimeoutMs = ATTACK_TIMEOUT_DEFAULT_MS;
 static uint32_t ThreadCount = ATTACK_THREADS_DEFAULT;
+static const char* Alpn = "h3-29";
+static uint32_t Version = QUIC_VERSION_DRAFT_29;
```

Comment:
FYI, defaulting to -29 since that's the most supported version out there right now.